ferry.go

```diff
@@ -686,6 +716,13 @@ func (f *Ferry) Run() {
 		}()
 	}
 
+	schemaFingerVerifierPrintWg := &sync.WaitGroup{}
```

Comment:
Could probably just reuse the `supportingServicesWg` instead of having another one here.